airbyte-integrations/connectors/source-genesys/source_genesys/schemas/analytics_conversations.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Make sure the schema follow the reference described here: https://docs.airbyte.com/connector-development/schema-reference